src/com/lftechnology/java/training/alina/collections/generics/vehiclelist/VehicleListDemo.java

```diff
@@ -0,0 +1,70 @@
+
+package com.lftechnology.java.training.alina.collections.generics.vehiclelist;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Write a program which consists of Generic class VehicleList . In this, list
+ * of cars brands and motorbike brands can be removed and added using a single
+ * generic class.
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class VehicleListDemo {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)